app/client/src/components/layers/layerOrder/LayerOrder.vue

```diff
@@ -106,6 +106,97 @@
             </v-layout>
           </v-card>
         </v-expansion-panel>
+        <v-expansion-panel
+          v-for="layer in ActivePois"
+          :key="layer.value"
+          class="layer-row"
+        >
+          <v-expansion-panel-header expand-icon="" v-slot="{}" class="handle">
+            <v-layout row class="pl-1" wrap align-center>
+              <v-flex class="checkbox" xs1>
+                <v-simple-checkbox
+                  :color="appColor.primary"
+                  :value="true"
+                  disabled
+                ></v-simple-checkbox>
+              </v-flex>
+              <v-flex xs10 class="light-text">
+                <h4 class="pl-2">
+                  {{
+                    layer.value === "hauptschule"
```

Comment:
I am uncertain why the pois.hauptschule_mittelschule is hardcoded here?

Reply:
it is because hauptschule didn't have any translation, so I though that hauptschule and mittelschule are written together 🤷‍♂️

Reply:
Right the translation was missing. I think now it should be fine.